packages/server/src/transaction-context/async-local-storage-transaction-context-provider.ts

```diff
@@ -0,0 +1,19 @@
+import { EvaluationContext } from '@openfeature/core';
+import { TransactionContext, TransactionContextPropagator } from './transaction-context';
+import { AsyncLocalStorage } from 'async_hooks';
```

Comment:
It looks like this is available as far back as Node v12.17.0 so I think it's fine to use. Originally, I thought we may need to lazy load when the class is instantiated.